functions/types.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	substraitgo "github.com/substrait-io/substrait-go"
 	"github.com/substrait-io/substrait-go/types"
+	"github.com/substrait-io/substrait-go/types/parameter_types"
```

Comment:
We should add the license identifier `// SPDX-License-Identifier: Apache-2.0` to this file

Reply:
added, Thanks